ca/server.go

```diff
@@ -569,9 +596,13 @@ func (s *Server) UpdateRootCA(ctx context.Context, cluster *api.Cluster) {
 	// ExternalCA interface that has different implementations for
 	// different CA types. At the moment, only CFSSL is supported.
 	var cfsslURLs []string
-	for _, ca := range cluster.Spec.CAConfig.ExternalCAs {
-		if ca.Protocol == api.ExternalCA_CAProtocolCFSSL {
-			cfsslURLs = append(cfsslURLs, ca.URL)
+	for _, extCA := range cluster.Spec.CAConfig.ExternalCAs {
+		urlsCACert := extCA.CACert
+		if extCA.CACert == nil {
+			urlsCACert = rCA.CACert // if there was no CA cert (it was nil), assume it's the original cert
```

Comment:
`rCA.CACert` is the root of trust we're generally using to validate TLS certificates.  `externalCACert` specifies which cert this external CA will use to sign - we want to match the external CA cert to the signing cert, which may be the new root cert if a root rotation is in progress.  I've renamed this variable and added a comment about why it's here.